downstream/modules/platform/con-gs-ansible-roles.adoc

```diff
@@ -2,14 +2,19 @@
 
 = Bundle content with Ansible roles
 
-A role is like a customized piece of automation content that bundles together relevant bits from automation playbooks to fit your system's specific needs. Roles are self-contained and portable, and can include groupings of tasks, variables, configuration templates, handlers, and other supporting files to orchestrate complicated automation flows. 
+A role is like a customized piece of automation content that bundles together relevant bits from playbooks to fit your system's specific needs. Roles are self-contained and portable, and can include groupings of tasks, variables, configuration templates, handlers, and other supporting files to orchestrate complicated automation flows. 
 
 Instead of creating huge playbooks with hundreds of tasks, you can use roles to break the tasks apart into smaller, more discrete units of work. 
 
 To learn more about roles, see link:https://www.redhat.com/en/topics/automation/what-is-an-ansible-role#why-use-a-role-instead-of-a-playbook[What is an Ansible Role-and how is it used?].
 
 == Creating a role
 
-You can create roles using the Ansible Galaxy CLI tool, which is included with your Ansible Automation Platform bundle. Access role-specific commands from the roles subcommand:
+You can create roles using the Ansible Galaxy CLI tool, which is included with your {PlatformNameShort} bundle. Access role-specific commands from the roles subcommand:
 
-//ADD CONTENT
+[source,bash]
+----
+ansible-galaxy role init <role_name>
+----
+
+Standalone roles outside of collections are still supported, but we recommend that you create new roles inside of a collection to take advantage of all the features {PlatformNameShort} has to offer.
```

Comment:
Style guidelines advise not to use "we recommend". Do not use to refer to IBM in technical information, as in "we recommend".

Suggest rewriting this to: 
Standalone roles outside of collections are still supported, but to take advantage of all the features {PlatformNameShort} as to offer, you need to create new roles inside of a collection.